js/components/ColorTag.js

```diff
@@ -0,0 +1,12 @@
+const React = require("react");
+
+class ColorTag extends React.Component {
+  render() {
+    const fontAttributes = this.props.attribute;
```

Comment:
I don't see an `attribute` property sent in App.js. If you want to get at the individual attributes, you refer to them by name, i.e. `this.props.color`